pkg/kubelet/apis/config/types.go

```diff
@@ -478,6 +478,11 @@ type KubeletConfiguration struct {
 	// If not specified the default value is ContainerRuntimeEndpoint
 	// +optional
 	ImageServiceEndpoint string
+
+	// PodresourcesLimits specifies the rate limits for the node-local podresources API endpoint
```

Comment:
My take is that to address the concern from the KEP we can have unconfigureable very high threshold to graduate the feature. It will protect from DDoS attacks, but will not be very limiting to scenarios.

If we will need configuration later we will need to create a common approach for all endpoints. Some sort of fair clients throttling may need to be implemented at that point.

Reply:
thanks @SergeyKanzhelev for chiming in. I implemented your suggestion reusing a subset of the code proposed here: https://github.com/kubernetes/kubernetes/pull/116459
I'll cose this PR if the sig prefer to move in the direction of the hardcoded defaults, evaluating a the common approach later.